src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/WaysWeSell.tsx

```diff
@@ -68,113 +66,72 @@ const data: StepsWithImageBlackDataType[] = [
-          <Box mx={[-2, -4]} px={[2, 4]} py={[4, 12]}>
-            <Text
-              mb={[1, 2]}
-              variant={["lg-display", "xl", "xxl"]}
-              color="white100"
-            >
-              A sales strategy tailored to your artwork
-            </Text>
+        <Box px={[2, 4]} py={[4, 12]}>
+          <Text mb={[1, 2]} variant={["lg-display", "xl", "xxl"]}>
+            A sales strategy tailored to your artwork
+          </Text>
+
+          <Text mb={[4, 4, 6]} variant={["xs", "sm"]}>
+            A dedicated specialist works with you to select the best sales
+            option for your artwork.
+          </Text>
+
+          <DesktopLayout />
 
-            <Text mb={[4, 4, 6]} variant={["xs", "sm"]} color="white100">
-              A dedicated specialist works with you to select the best sales
-              option for your artwork.
-            </Text>
-            <DesctopLayout />
-            <MobileLayout />
-          </Box>
-        </HorizontalPadding>
+          <MobileLayout />
+        </Box>
       </AppContainer>
     </FullBleed>
   )
 }
 
-const ShelfItem: React.FC<StepsWithImageBlackDataType> = ({
-  src,
-  title,
-  text,
-  srcSet,
-}) => {
-  const { width } = useWindowSize()
-
-  const aspectRatio = BIG_IMAGE_HEIGHT / IMAGE_WIDTH
-
-  // calculating the width of the image contailer
-  // (screen width - HorizontalPadding - px of the parent - gridColumnGap) / 3
-  const collumnHeightMediumcreen =
-    ((width - 20 * 2 - 20 * 2 - 20 * 2) / 3) * aspectRatio
-  const collumnHeightBigScreen =
-    ((width - 40 * 2 - 40 * 2 - 40 * 2) / 3) * aspectRatio
-
+const ShelfItem: React.FC<ShelfItemProps> = ({ image, title, text }) => {
   return (
-    <>
-      <Box
+    <Stack gap={[1, 2]}>
+      <ResponsiveBox
+        aspectWidth={image.width}
+        aspectHeight={image.height}
         maxWidth="100%"
-        bg="black100"
-        mb={[1, 2]}
-        height={["auto", collumnHeightMediumcreen, collumnHeightBigScreen]}
-        maxHeight={BIG_IMAGE_HEIGHT}
       >
-        <Flex height="100%" alignItems="flex-end">
-          <Image
-            src={src}
-            srcSet={srcSet}
-            width="100%"
-            height="auto"
-            lazyLoad
-            alt={`${title} image`}
-            style={{
-              display: "flex",
-              alignSelf: "flex-end",
-            }}
-          />
-        </Flex>
-      </Box>
-
-      {title && (
-        <Text
-          mb={[0.5, 1]}
-          variant={["md", "lg-display", "xl"]}
-          color="white100"
-        >
-          {title}
-        </Text>
-      )}
+        <Image
+          {...resized(image.src, { width: 500, height: 500 })}
+          width="100%"
+          height="100%"
+          lazyLoad
+          style={{ display: "block" }}
+        />
+      </ResponsiveBox>
+
+      <Stack
+        gap={[0.5, 1]}
+        // Magic numbers to bottom align images
+        minHeight={[100, 200, 140]}
```

Comment:
This _should_ be structured so that we do this in 2 rows, eliminating the need for any manual adjustment. But this is fine for now.